Model/Plugin/Order/AddressSave/Edit.php

```diff
@@ -30,8 +30,9 @@
  */
 class Edit
 {
+    const TRANSACTION_ID = 'transaction_id';
     const CHARGE_ID = 'charge_id';
-    const API_CHARGES_PATH = '/api/v2/charges/';
+    const API_TRANSACTIONS_PATH = '/api/v1/transactions/';
```

Comment:
why not use the same constant from `Gateway/Helper/Request/Action.php` or even better move them into a `Constants.php` file which allows us to reuse instead of creating the same constants across multiple files. This will make our lives easier when we need to update them in the future.